src/main/java/uk/gov/hmcts/reform/pdf/service/endpoint/v2/PDFGenerationEndpointV2.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.pdf.service.endpoint.v2;
+
+import io.swagger.annotations.Api;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.pdf.generator.HTMLToPDFConverter;
+import uk.gov.hmcts.reform.pdf.service.domain.GeneratePdfRequest;
+
+@Api
+@RestController
+@RequestMapping(
+    path = "pdf",
```

Comment:
should probably be "/pdfs"
https://hmcts.github.io/restful-api-standards/#134

Reply:
:+1: